seatunnel-connectors-v2/connector-cdc/connector-cdc-base/pom.xml

```diff
@@ -30,6 +30,7 @@
 
     <properties>
         <hikaricp.version>4.0.3</hikaricp.version>
+        <junit.vserion>4.13.2</junit.vserion>
```

Comment:
Why not use junit 5.9.0 define in root pom.xml?

Reply:
Currently, a large number of tests are used for this version